test/cpp_api_parity/torch_nn_modules.py

```diff
@@ -148,8 +148,14 @@
     'AlphaDropout': TorchNNModuleMetadata(),
     'Embedding': TorchNNModuleMetadata(),
     'EmbeddingBag': TorchNNModuleMetadata(),
-    'CosineSimilarity': TorchNNModuleMetadata(),
-    'PairwiseDistance': TorchNNModuleMetadata(),
+    'CosineSimilarity': TorchNNModuleMetadata(
+        cpp_default_constructor_args="()",
+        num_attrs_recursive=2,
+    ),
+    'PairwiseDistance': TorchNNModuleMetadata(
+        cpp_default_constructor_args="()",
+        num_attrs_recursive=3,
+    ),
```

Comment:
Thanks for updating the entries here! The parity test mechanism is still a bit fragile and I think it's okay to leave the entries unchanged here for now (mostly because I don't want contributors to have to worry about parity test ;)). I will add the parity test in a separate PR after we merge this PR.